### ISSUE TITLE
Issue when processing space-separated data

### ISSUE THREAD

**Issue by DavyCao** [modin-project/modin#311]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **Modin installed from (source or binary)**: pip3 install modin
- **Modin version**: 0.2.5
- **Python version**: 3.5.2
- **Exact command to reproduce**: df = pd.read_csv(filename, header=None, delim_whitespace=True)

### Describe the problem

I have a space-separated data file.
Using pandas, I can separate the values:

```
Processing file: data/data_1_1_36598.csv
                  0                     1                 2   3           4           5    6           7
0           0.169881  192.168.2.2:36598  192.168.2.3:80  32  0xdb0d77fb  0xdb0d44fb   10  2147483647 

```

But when using modin, the values are not separated into different columns:
```
Processing file: data/data_1_1_36598.csv
                                                         0
0        0.169881371 192.168.2.2:36598 192.168.2.3...
```

**Comment by devin-petersohn** [comment]
HI @DavyCao thanks for posting this. I am able to reproduce this with the values you provided. 

It works for some datasets, but not others. This is an interesting one, I will get it fixed. Thanks again for finding this!

**Comment by devin-petersohn** [comment]
@DavyCao I just commited the fix for this in #314. If you'd like to try it right now, you install via the github (docs on how to do that [here](https://modin.readthedocs.io/en/latest/installation.html#installing-from-the-github-master-branch))
